airbyte-integrations/connectors/source-gong/pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = [ "poetry-core>=1.0.0",]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "0.1.5"
+version = "0.1.6"
```

Comment:
There's also `dockerImageTag` in `metadata.yaml` that would need a version bump as well — can you add that please?

Reply:
will do

Reply:
just updated this, thanks for the heads up!